control-plane/pkg/prober/async_prober_test.go

```diff
@@ -297,9 +297,9 @@ func TestAsyncProberRotateCACerts(t *testing.T) {
 	}
 
 	t.Run("one pod - TLS certs before rotation", func(tt *testing.T) {
-		require.Eventuallyf(t, probeFunc, 5*time.Second, 100*time.Millisecond, "")
-		require.Eventuallyf(t, func() bool { return wantRequestCountMin.Load() == 1 }, 5*time.Second, 100*time.Millisecond, "got %d, want 1", wantRequestCountMin.Load())
-		require.Eventuallyf(t, func() bool { return wantRequeueCountMin.Load() == 1 }, 5*time.Second, 100*time.Millisecond, "got %d, want 1", wantRequeueCountMin.Load())
+		require.Eventuallyf(tt, probeFunc, 5*time.Second, 250*time.Millisecond, "")
```

Comment:
Basically, `t` is the parent test, while `tt` is the child test. So, one of the errors being reported was that a child test was failing the parent rather than itself.